wandb/sdk/internal/tb_watcher.py

```diff
@@ -123,7 +123,7 @@ def __init__(
         self._force = force
         # TODO(jhr): do we need locking in this queue?
         self._watcher_queue = queue.PriorityQueue()
-        wandb.tensorboard.reset_state()
+        wandb.tensorboard.reset_state()  # type: ignore
```

Comment:
why are types ignored?

Reply:
same, I don't want to declare a module-level public api that we'll remove in 0.19. why pollute?
also, we moved all these integrations to `wandb.integrations`, `wandb.tensorboard` was kept for compatibility as too many people have it hardcoded.